pkg/scheduler/framework/plugins/interpodaffinity/plugin.go

```diff
@@ -52,6 +55,74 @@ func (pl *InterPodAffinity) Name() string {
 	return Name
 }
 
+func (pl *InterPodAffinity) Requeue(ctx context.Context, p *v1.Pod, event framework.ClusterEvent, oldObj, obj interface{}) framework.SchedulingHint {
+	if !event.In(pl.EventsToRegister()) {
+		// not interested event.
+		return framework.PodNotAffected
+	}
+
+	switch event.Resource {
+	case framework.Pod:
+		return pl.requeueByPodEvent(ctx, p, event, oldObj, obj)
+	case framework.Node:
+		// TODO(sanposhiho): implement it
+		return framework.PodMaybeSchedulable
+	}
+
+	return framework.PodNotAffected
+}
+
+func (pl *InterPodAffinity) requeueByPodEvent(ctx context.Context, p *v1.Pod, event framework.ClusterEvent, oldObj, obj interface{}) framework.SchedulingHint {
```

Comment:
Should probably accept a logger instead of a context. There shouldn't be a need for a context because this shouldn't call any clients.

Reply:
> because this shouldn't call any clients.

It does make sense.